devtools/datasette/fly/run.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+set -eux
+
+shopt -s nullglob
+
+find /data/ -name '*.sqlite' -delete
+mv all_dbs.tar.zst /data
```

Comment:
When are the tar files mounted or copied to the docker container?

Reply:
The compression step dumps the output into the docker context, which then gets copied in `COPY . /app`. Unfortunately we can't get around copying into the image, for now - fly.io doesn't support a "pre-populate the volume with `scp`, then deploy" pattern yet.